tests/integration/stepfunctions/templates/services/lambdafunctions/return_bytes_str.py

```diff
@@ -0,0 +1,2 @@
+def handler(event, context):
+    return b'"HelloWorld!"'
```

Comment:
Maybe we should also test with different values here (list, boolean, integer, float, string, bytes, dict)

Reply:
Absolutely, thank you:) Given this i added more fixes to this pr about default values for Lambda Task Payloads and start_execution inputs. I added these test scenarios without declaring new lambda functions as other json primitives can be passed to sfn as json values (not bytes).